cocos/rendering/custom/graph.ts

```diff
@@ -156,7 +156,7 @@ export class OutEPI implements IterableIterator<EPD> {
         this.iterator = iterator;
         this.source = source;
     }
-    [Symbol.iterator] () {
+    [Symbol.iterator] (): this {
```

Comment:
`this` is a valid type?

Reply:
it's valid type in TS, but it's better to mark it as the typeof this